<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rockchip_gpio_calculate v0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);
border-width:2px;
border-style:solid;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="InfoGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 237);
border-width:2px;
border-style:solid;
border-radius: 10px;
border-color:rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="BTVersionlabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>41</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QLabel" name="DeviceVersionlabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>191</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>GPIO</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gpio_num">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>101</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>17</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="gpio_num2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>111</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>17</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="gpio_output">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>331</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="gpio_num3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>17</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="DeviceVersionlabel_5">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>60</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </widget>
    <zorder>gpio_output</zorder>
    <zorder>gpio_num</zorder>
    <zorder>gpio_num2</zorder>
    <zorder>groupBox</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
